addons/rating/static/src/components/thread_needaction_preview/thread_needaction_preview.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.ThreadNeedactionPreview" t-inherit-mode="extension">
+        <xpath expr="//*[hasclass('o_ThreadNeedactionPreview_messageBody')]" position="replace">
+            <t t-if="thread.lastNeedactionMessageAsOriginThread and thread.lastNeedactionMessageAsOriginThread.ratingImg">
+                <span class="o_ThreadNeedactionPreview_rating_text">Rating:</span>
+                <span class="o_ThreadNeedactionPreview_rating_image">
+                    <img height="17px" t-att-src="thread.lastNeedactionMessageAsOriginThread.ratingImg"/>
```

Comment:
Where is that height coming from? It should be added in scss, if possible depending on an existing variable.